data/hai missions.txt

```diff
@@ -1053,3 +1053,250 @@ mission "Hai Honeymoon"
+	on complete
+		event "Nanachi: payment" 180
+		conversation
+			`Landing back on Allhome, the festival group gives you and Nanachi a warm hug, "We sold tens of thousands of tickets for the festival by promising to host a great art exhibition with pieces from all over human space, without them it'd be a complete failure, and we'd be in serious trouble with our sponsors." Narancia says.`
+			`	As the cargo is rushed to the exhibition site, Nanachi shows you the chip she got as payment, and the balance of her bank account: this last payment allowed her to fully pay off her loan. "Captain," she speaks with some difficulty, and tries to hide her face, as if to wipe away some tears. "I'll never be able to thank you properly, if not for you I'd sill be lost in that spaceport, and would eventually have to go back to my mining job. I'll work hard now, I'll save up and pay you for everything!" She hugs you and bows to you in a way you'd see in movies, as if trying to immitate the gesture. You two finally say goodbye, and head back to your ships.`
+
+
+
+event "Nanachi: payment"
+
+
+
+mission "Nanachi: Payment"
+	landing
+	source
+		government "Hai"
+		not planet "Darkwaste"
+	to offer
+		has "event: Nanachi: payment"
+	on offer
+		payment 700000
+		event "Nanachi: Update" 720
+		conversation
+			`Shortly after you land, you get a message from Nanachi, the young hai pilot who you helped start out. "Dear captain <last>, I hope you're doing well! As I promised, ever since we parted ways I've been working towards getting enough money to pay you for being so kind to help me. I've sent you <payment>. I'm deeply grateful to you for helping make my dream come true."`
+				decline
+
+
+
+event "Nanachi: Update"
+
+
+
+mission "Nanachi Meeting"
+	landing
+	source
+		government "Hai"
+		not planet "Darkwaste"
+	to offer
+		has "event: Nanachi: Update"
+		random > 30
+	on offer
+		conversation
+			`After you leave your ship and start making your way through the docking area, you notice a Hai running towards you and waving. As they get closer, you realize that it's Nanachi, the Hai pilot who you helped out years ago. "I was just about to go back to my crew when I saw you coming out of your ship. I know a nice snack bar here, let's go there!" Before you can even say hello she's dragged you to the bar, and ordered some drinks for the two of you. She tells you how the last years have been, and of her experiences as a captain. She has long since sold her old ship, and now pilots a Geocoris, transporting large amounts of cargo across Hai space. You also tell her about how your life has been ever since you last met, of your adventures and how far you've come. You keep swapping stories for hours, until the bar is ready to close.`
```

Comment:
`She tells you how the last years have been, and of her experiences as a captain` -> `She tells you about her last few years as a captain`